pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -439,22 +596,40 @@ We understand this is tedious and are working on a better solution for setting u
     </button>
     <div class="collapsible-content collapsed">
         <div class="card-content">
-            While it's technically possible to edit <code>config-default.yml</code> to match your server, it is heavily discouraged.
+            While it's technically possible to edit the values in <code>constants.py</code> to match your server, it is heavily discouraged.
             This file's purpose is to provide the configurations the Python bot needs to run in the Python server in production, and should remain as such.
-            In contrast, the <code>config.yml</code> file can remain in your local copy of the code, and will be ignored by commits via the project's <code>.gitignore</code>.
+            In contrast, the <code>.env.server</code> file can remain in your local copy of the code, and will be ignored by commits via the project's <code>.gitignore</code>.
         </div>
     </div>
 </div>
 <br>
 
-#### .env
+##### .env
 The second file you need to create is the one containing the environment variables, and needs to be named `.env`.
 Inside, add the line `BOT_TOKEN=YourDiscordBotTokenHere`. See [here](../creating-bot-account) for help with obtaining the bot token.
 
-The `.env` file will be ignored by commits.
+**Note**: The `.env` is and should remain ignored by git, otherwise you risk pushing sensitive information.
```

Comment:
You do need a guild id in your env, since there's the `Guild` class that needs to pick it up.

Also, if you we go with the above suggestion about where we should place `.env` & `.env.server`, i'll mention it there

Reply:
Unresolving since the comment is still relevant.

You have this part:
https://github.com/python-discord/site/blob/493ab3c7381ba89f640b7b53d2d0d96bf8944dbf/pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md?plain=1#L69-L78

And then you have this part:
https://github.com/python-discord/site/blob/493ab3c7381ba89f640b7b53d2d0d96bf8944dbf/pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md?plain=1#L194-L196

The second part is not up to date, the up-to-date version is the first part. Since this is relevant to both automatic and manual config, and it's the same information, it should just be combined, and appear above both sections, probably here:
https://github.com/python-discord/site/blob/493ab3c7381ba89f640b7b53d2d0d96bf8944dbf/pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md?plain=1#L53-L55

I suggest having it before the `.env.server` explanation here:
https://github.com/python-discord/site/blob/493ab3c7381ba89f640b7b53d2d0d96bf8944dbf/pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md?plain=1#L45

And then the `#### Automatic configuration` and `#### Manual configuration` will be under the `.env.server`, as they should. In that case those two `####` parts don't need their own `.env.server` sections under them, as they're entirely about the .env.server.